### PR TITLE
migrate from dep to Go modules

### PR DESCRIPTION
**Pull request by kopiczko**
This is from @tomahawk28. 

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+go 1.13
+
+require (
+	github.com/DATA-DOG/go-sqlmock v1.4.1 // indirect
+	github.com/Masterminds/goutils v1.1.0 // indirect
+	github.com/Masterminds/semver v1.5.0
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/dsnet/compress v0.0.1 // indirect
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/errors v0.1.0
```
Comment (wtty-fool) [inline go.mod]: Should be `v0.2.0`. We need to merge https://github.com/giantswarm/errors/pull/33

**Comment by tomahawk28** [comment]
cc> @kopiczko

File: go.mod (hunk 1 above)
Comment (tomahawk28) [reply go.mod]: Updated, @wtty-fool PTAL

**Comment by kopiczko** [comment]
@wtty-fool: using orb should be a separate PR.

@tomahawk28: whats up?

**Pull request merged**